/laser_processing_node:
  ros__parameters:
    max_dist: 200.0
    min_dist: 3.0
    scan_line: 64
    scan_period: 0.1
    vertical_angle: 2.0

/odom_estimation_node:
  ros__parameters:
    map_resolution: 0.4
    max_dist: 200.0
    min_dist: 3.0
    scan_line: 64
    scan_period: 0.1
    vertical_angle: 2.0

/laser_mapping_node:
  ros__parameters:
    map_resolution: 0.4
    max_dist: 200.0
    min_dist: 3.0
    scan_line: 64
    scan_period: 0.1
    vertical_angle: 2.0
